notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -36,68 +130,130 @@
+    "    contribution[\"dir\"] = contribution_dir\n",
     "\n",
-    "SDG_OUTPUT_DIR = WORKSPACE_DIR / \"sdg\"\n",
-    "SDG_OUTPUT_DIR.mkdir(exist_ok=True)"
+    "    for subdir in [\"source_documents\", \"conversion\", \"chunking\", \"authoring\"]:\n",
+    "        (contribution_dir / subdir).mkdir(parents=True, exist_ok=True)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "344b7ac5-fc2a-40a8-8e1f-e8dd8b1153e7",
    "metadata": {},
    "source": [
-    "## Document Conversion\n",
+    "## Data Gathering\n",
     "\n",
-    "This notebook uses [Docling](https://github.com/docling-project/docling) to convert any type of document into a Docling Document. A Docling Document is the representation of the document after conversion that can be exported as JSON. The JSON output of this notebook can then be used in others such as one that uses Docling's chunking methods."
+    "The files for each of the contributions should be copied into `WORKSPACE_DIR/<CONTRIBUTION NAME>/source_documents` directory for the following conversion step to automatically find them."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "b91d4b2e-19cd-46e7-a912-ba9b2904c7cd",
+   "id": "26501e2f-7215-441f-9efa-075f87024893",
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -qq docling"
+    "import shutil\n",
+    "\n",
+    "# Inference Time Scaling Contribution\n",
+    "orig_path = Path(\"sample-pdfs/inference-time-scaling.pdf\")\n",
+    "dst_path = WORKSPACE_DIR / contribution_name / \"source_documents\"\n",
+    "\n",
+    "shutil.copy(orig_path, dst_path)\n",
+    "\n",
+    "# NFL Rules Contribution\n",
+    "rules_2022 = Path(\"sample-pdfs/2022-nfl-rulebook.pdf\")\n",
+    "rules_2023 = Path(\"sample-pdfs/2023-nfl-rulebook.pdf\")\n",
+    "rules_dst = WORKSPACE_DIR / contribution2_name / \"source_documents\"\n",
+    "\n",
+    "shutil.copy(rules_2022, rules_dst)\n",
+    "shutil.copy(rules_2023, rules_dst) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68478061",
+   "metadata": {},
+   "source": [
+    "Make sure to review this list of files to make sure all of the files you're expecting is in each of the contributions."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "3f3804ef-4961-44b1-91c9-62929f422702",
+   "id": "5325fa71-d09f-457f-9e55-be106dcf78e0",
    "metadata": {},
    "outputs": [],
    "source": [
-    "files = []\n",
-    "\n",
-    "if SOURCE_DOCUMENT:\n",
-    "    files.append(Path(SOURCE_DOCUMENT))\n",
-    "else:\n",
-    "    print(\"***** WARNING! Only one file at a time is supported at this time.\")\n",
-    "    files = list(SOURCE_DOCUMENT_DIR.rglob(\"*.pdf\"))\n",
-    "    print(f\"***** Using {files[0]})\")\n",
+    "print(f\"Files to pre-process\\n--------------------\")\n",
+    "for contribution in contributions:\n",
+    "    print(f\"\\nContribution: {contribution[\"name\"]}\")\n",
+    "    print(f\"Files:\")\n",
```

Comment:
```suggestion
    "    print(f\"\\nContribution: {contribution.get('name')}\")\n",
    "    print(\"Files:\")\n",
```
some nitpicks:
- using `.get` is safer since it return `None` if the key does not exist instead of raising an error